monkey/monkey_island/cc/services/zero_trust/monkey_details_service.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
+
+from bson import ObjectId
+
+from monkey_island.cc.models.zero_trust.monkey_finding_details import MonkeyFindingDetails
+
+# How many events of a single finding to return to UI.
+# 50 will return 50 latest and 50 oldest events from a finding
+EVENT_FETCH_CNT = 50
+
+
+class MonkeyDetailsService:
+
+    @staticmethod
+    def fetch_details_for_display(finding_id: ObjectId):
+        pipeline = [{'$match': {'_id': finding_id}},
+                    {'$addFields': {'oldest_events': {'$slice': ['$events', EVENT_FETCH_CNT]},
+                                    'latest_events': {'$slice': ['$events', -1 * EVENT_FETCH_CNT]},
+                                    'event_count': {'$size': '$events'}}},
+                    {'$unset': ['events', 'checked_subnet_pairs']}]
+        details = list(MonkeyFindingDetails.objects.aggregate(*pipeline))
+        if details:
+            details = details[0]
```

Comment:
this is a bad practice since details changes its meaning (and type). Use a different variable name for `details[0]`